ArchiSteamFarm/Commands.cs

```diff
@@ -2852,20 +2853,22 @@ public sealed class Commands {
 				return FormatBotResponse(Strings.BotNotConnected);
 			}
 
-			HashSet<Steam.Asset> inventory = await Bot.ArchiWebHandler.GetInventory(Bot.SteamID, wantedTypes: new HashSet<Steam.Asset.EType> { Steam.Asset.EType.BoosterPack }).ConfigureAwait(false);
-
-			if ((inventory == null) || (inventory.Count == 0)) {
-				return FormatBotResponse(string.Format(Strings.ErrorIsEmpty, nameof(inventory)));
-			}
-
 			// It'd make sense here to actually check return code of ArchiWebHandler.UnpackBooster(), but it lies most of the time | https://github.com/JustArchi/ArchiSteamFarm/issues/704
 			bool completeSuccess = true;
 
 			// It'd also make sense to run all of this in parallel, but it seems that Steam has a lot of problems with inventory-related parallel requests | https://steamcommunity.com/groups/ascfarm/discussions/1/3559414588264550284/
-			foreach (Steam.Asset item in inventory) {
-				if (!await Bot.ArchiWebHandler.UnpackBooster(item.RealAppID, item.AssetID).ConfigureAwait(false)) {
-					completeSuccess = false;
+			try {
+				await foreach (Steam.Asset item in Bot.ArchiWebHandler.GetInventoryAsync(Bot.SteamID).Where(x => x.Type == Steam.Asset.EType.BoosterPack).ConfigureAwait(false)) {
```

Comment:
I'd rename `item` would be an easier understandable name instead of `x`

Reply:
Yeah, I don't like those short forms either, even when in LINQ, you always evaluate over something, it's better if we know what that something is.

Reply:
Even if the names clashed (which they don't), alternatives like "asset" would be better than a plain x

Reply:
I tried to name every parameter as readable as possible, but this one somehow slipped through, thanks!

Reply:
Sometimes you can go with the short form when you're explicit about that earlier, for example in `catch (Exception e)`, you really don't need to know that `e` is `exception`, you have it outright stated.

But here you do not, so it's better to use explicit name.